netaddr.go

```diff
@@ -86,6 +86,38 @@ func ParseIP(s string) (IP, error) {
 	return IP{v6}, nil
 }
 
+// IPAddr returns the net.IPAddr representation of an IP. If the IP contains a
+// zone identifier, the Zone field of the returned IPAddr is populated.
```

Comment:
document that the returned value is always non-nil, but the IPAddr.IP will be nil if ip is the zero value?